specs/registry-auth.md

```diff
@@ -0,0 +1,143 @@
+# Registry Authentication
+
+Registry access is required for pulling the manifests of the images to be verified along with their signatures as well as other advanced operations. This documentation specifies how authentication to the remote registry works and how registry credentials are stored.

```

Comment:
Suggesting some rewording, feel free to modify further.

```suggestion
Registry access is required during signature generation and signature verification workflows. Based on the workflow, write access is required to push signatures, and read access is required to pull manifests, and signatures associated with a given manifest. This documentation specifies how authentication to the remote registry works and how registry credentials are stored.

```

Reply:
Registry access is not a MUST during signature generation and signature verification workflows, especially for local signing scenarios.